sdk/aqueduct/utils.py

```diff
@@ -197,6 +199,13 @@ def serialize_function(
         with open(os.path.join(dir_path, MODEL_PICKLE_FILE_NAME), "wb") as f:
             pickle.dump(func, f)
 
+        # Write function source code to file
+        source_file = "{}.py".format(op_name)
+        with open(os.path.join(dir_path, source_file), "w") as f:
+            source = inspect.getsource(func)
+            print(source)
```

Comment:
is this print supposed to be here? If it is, was should use the logger.

Reply:
Removed